docs/arrays.md

```diff
@@ -59,6 +59,10 @@ By default, when a task fails the computation of job continues.
 You can change it by ``--max-fails=X`` where ``X`` is non-negative integer.
 If more tasks then ``X`` fails, then the rest of non-finished tasks are canceled.
 
+## Time limit
+
+Time limit setted by ``--time-limit`` is a time limit for each task separatatelly.
```

Comment:
You can set a time limit for each task separately with the ``--time-limit`` option.

Reply:
My formulation is bad but your wording suggest that you can set different time limit for each task